crates/storage-api/src/invocation_status_table/mod.rs

```diff
@@ -550,15 +564,23 @@ impl CompletedInvocation {
     }
 }
 
+#[derive(Debug, Clone, Eq, PartialEq)]
+pub struct InvokedOrKilledInvocationStatusLite {
+    pub invocation_id: InvocationId,
+    pub invocation_target: InvocationTarget,
+    /// If true, original status is Invoked, otherwise is Killed
+    pub is_invoked: bool,
```

Comment:
I think this is fine for the time being, this package is crowded with too many types. I tried to add some enum discriminator already, and it didn't end well naming wise. 

Eventually I wanna simply rename `InvocationStatus` to `Invocation`, and have an `InvocationStatus` enum which is just the discriminants, and then here use that enum. I can pull this through in a subsequent PR, but it's a big change and i don't wanna do it now to avoid rebase hell.